src/messageComposer/middleware/textComposer/textMiddlewareUtils.ts

```diff
@@ -134,3 +134,13 @@ export const getTokenizedSuggestionDisplayName = ({
       : [displayName],
   },
 });
+
+export const isTextMatched = (input: string, command: string): boolean => {
```

Comment:
Isn't this a duplicate logic already present in another util function in this file?

Reply:
Which utility are you talking about here?